prow/OWNERS

```diff
@@ -3,3 +3,4 @@ approvers:
 - kargakis
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 6: could not find expected ':'.